packages-exp/app-check-exp/src/public-types.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FirebaseApp } from '@firebase/app-exp';
+import { CustomProvider, ReCaptchaV3Provider } from './providers';
+
+/**
+ * The Firebase AppCheck service interface.
+ *
+ * @public
+ */
+export interface AppCheck {
+  /**
+   * The FirebaseApp this Functions instance is associated with.
```

Comment:
Should this be "`FirebaseApp` object" (a literal) or just Firebase app?

Also, recommend "Cloud Functions instance" unless, again, it is a literal (which would be backticked).

Reply:
Thank you, that was copied and pasted from the Functions repo and should be AppCheck! Also made them both literals.